services/web/server/src/simcore_service_webserver/notifications/_rabbitmq_exclusive_queue_consumers.py

```diff
@@ -90,7 +90,7 @@ async def _progress_message_parser(app: web.Application, data: bytes) -> bool:
         ).to_socket_dict()
 
     if message:
-        await send_message_to_user(
```

Comment:
do we still use this type of notification to user_id? if not, we should remove it to avoid this type of errors.